code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -377,7 +377,7 @@
 					if(SSshuttle.emergency.timeLeft(1) > initial(SSshuttle.emergencyCallTime)*0.5)
 						SSticker.mode.make_antag_chance(humanc)
 
-	if(humanc && CONFIG_GET(flag/roundstart_traits))
+	if(job.job_flags & JOB_ASSIGN_QUIRKS && humanc && CONFIG_GET(flag/roundstart_traits))
```

Comment:
```suggestion
	if((job.job_flags & JOB_ASSIGN_QUIRKS) && humanc && CONFIG_GET(flag/roundstart_traits))
```
Same here, feel free to disregard otherwise.

Reply:
no i liked this